activerecord/lib/active_record/railtie.rb

```diff
@@ -235,5 +235,13 @@ class Railtie < Rails::Railtie # :nodoc:
         end
       end
     end
+
+    initializer "active_record.set_filter_attributes" do
+      config.after_initialize do
```

Comment:
This is unnecessary, and in fact it causes a problem: if the user sets `filter_attributes` in an initializer, their configuration will be overwritten by `filter_parameters` and lost.

It might be worth adding a test for this case.

Reply:
Thanks for your valuable advice, I just wonder is it necessary for user to set `filter_attributes` in an initializer? Since user could set `filter_attributes` in `ApplicationRecord`  e.g.
```ruby
class ApplicationRecord < ActiveRecord::Base
  self.abstract_class = true
  self.filter_attributes = [:credit_card_number]
end
```
Otherwise, I should add a new instance variable `@filter_attributes` in `Rails::Application::Configuration`, please correct me if there is any mistake, thanks 😉

Reply:
Any option on `ActiveRecord::Base` can be configured by setting a key on `Rails.application.config.active_record`. I was thinking of someone adding an initializer that mirrors the one for `filter_parameters`, since they're so similar:

https://github.com/rails/rails/blob/e6ba30efbf107954db2707410d78d77680d69995/railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb.tt